pkgs/tools/admin/pulumi-packages/pulumi-language-python.nix

```diff
@@ -36,10 +27,4 @@ buildGoModule rec {
   nativeCheckInputs = [
     python3
   ];
-
```

Comment:
This breaks Python stacks (because `pulumi-language-python-exec` is missing). The following fixes it:

```nix
  postInstall = ''
    cp ../pulumi-language-python-exec           $out/bin
    cp ../../dist/pulumi-resource-pulumi-python $out/bin
    cp ../../dist/pulumi-analyzer-policy-python $out/bin
  '';
```

Reply:
Thanks, added it!